azure-pipelines/azure-pipelines.yml

```diff
@@ -521,6 +523,17 @@ stages:
           --build-arg PROJECT_ROOT="$(System.DefaultWorkingDirectory)" $(System.DefaultWorkingDirectory) \
           -t edgehub/deviceshifu-http-http:$(release_tag) --push
         displayName: build edgehub/deviceshifu-http-http
+      - task: Docker@2
+        displayName: Login to DockerHub
+        inputs:
+          command: login
+          containerRegistry: cn-dockerhub-connection
+      - script: |
+          docker buildx build --platform=linux/amd64,linux/arm64,linux/arm \
+          -f $(System.DefaultWorkingDirectory)/dockerfiles/Dockerfile.deviceshifuHTTP \
+          --build-arg PROJECT_ROOT="$(System.DefaultWorkingDirectory)" $(System.DefaultWorkingDirectory) \
+          -t $(cn-docker-registry)/shifu/deviceshifu-http-http:$(release_tag) --push
+        displayName: build edgehub/deviceshifu-http-http
```

Comment:
`edgehub` => `$(cn-docker-registry)/shifu/deviceshifu-http-http`, same for all other occurrences